src/extension/vscode.proposed.chatProvider.d.ts

```diff
@@ -5,14 +5,6 @@
 
 declare module 'vscode' {
```

Comment:
You should set the api version too for changes like this. Although note that going from no version to a version means that old extensions would still be considered compatible.

Reply:
I'm not super well versed in how the new api version stuff works. I still bump the engine then too?

Reply:
Pushed a couple PRs to add api versions. Engine version is fine